### ISSUE TITLE
interleaved encoded

### ISSUE THREAD

**Issue by czh-rot** [ranran0523/SPECNN#2]
I apologize for the repeated disturbance.  This time, I would like to inquire whether your code has been fixed and made public.  This is mainly because I have some confusion about the description in your paper.  If the code has not been fixed yet, could you please help me clarify my current confusion?  The main confusion arises from Figure 3 in the paper, where, for HE-group Conv, the number of convolutional kernels seems to decrease.  For example, compared to the general HE Conv, K13 and K14 disappear.  Therefore, I find it challenging to understand how the output of HE-group Conv is computed based on this figure. 
![image](https://github.com/ranran0523/SPECNN/assets/55588701/337d1190-e56b-4268-b1c5-bc37ec09590a)

**Comment by ranran0523** [comment]
Yes, the convolutional kernel would be decreased by using the group-convolution layers. It works by cutting the connections from input channels to output channels.  In general convolution, Kernel K13 and K14 are computed with input channel 1 to generate output channel 3 & 4 (e.g. ch1xK13+ch2xK23+ch3xK33+ch4xK43=out3). However, we have input channels 1 & 2 in a group, channels 3 & 4 in another, in which out3=ch3xK33+ch4xK43. You can also refer the attached figure to see how group convolution works.
![group_convolution](https://github.com/ranran0523/SPECNN/assets/58201595/3001d150-9c46-4c4c-996b-9d4869602fd1)

**Comment by czh-rot** [comment]
> Yes, the convolutional kernel would be decreased by using the group-convolution layers. It works by cutting the connections from input channels to output channels. In general convolution, Kernel K13 and K14 are computed with input channel 1 to generate output channel 3 & 4 (e.g. ch1xK13+ch2xK23+ch3xK33+ch4xK43=out3). However, we have input channels 1 & 2 in a group, channels 3 & 4 in another, in which out3=ch3xK33+ch4xK43. You can also refer the attached figure to see how group convolution works. ![group_convolution](https://user-images.githubusercontent.com/58201595/282310950-3001d150-9c46-4c4c-996b-9d4869602fd1.png)
thanks for the reply!
